apps/portals/src/portal-components/RouteControl.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useLayoutEffect, useRef } from 'react'
+import {
+  Tab,
+  Tabs,
+  TabScrollButton,
+  TabScrollButtonProps,
+  useMediaQuery,
+  useTheme,
+} from '@mui/material'
+
+export type RouteControlProps = {
+  handleChanges: (text: string, index: number) => void
+  isSelected: (name: string) => boolean
+  customRoutes: string[]
+}
+
+function CustomScrollButton(props: TabScrollButtonProps) {
+  if (props.disabled) {
+    return <></>
+  }
```

Comment:
Render nothing if the button is disabled; this is necessary to get the left alignment right if these scroll controls are visible.